oelint_adv/rule_base/rule_tasks_doc_strings.py

```diff
@@ -16,7 +16,7 @@ def __init__(self) -> None:
     def check(self, _file: str, stash: Stash) -> List[Tuple[str, int, str]]:
         res = []
         for item in stash.GetItemsFor(filename=_file, classifier=Function.CLASSIFIER):
-            if item.FuncName in CONSTANTS.FunctionsKnown or item.FuncName in ['', 'anonymous']:
+            if item.FuncName in CONSTANTS.FunctionsKnown or item.FuncName in ['', '__anonymous']:
```

Comment:
Fails in test for exactly what the patch tries to fix

Reply:
I see what is happening here the `__` prefix of  `__anonymous` is mistakenly taken as two override delimiter, due to the lack of any other evidence.

```
A:abc = "1"

python __anonymous () {
    print("Hello world")
}
```

works with your code fix, but 

```
python __anonymous () {
    print("Hello world")
}
```

not.
I think the proper fix would be probe for both `anonymous` and `__anonymous`.
Also please add some more tests to https://github.com/priv-kweihmann/oelint-adv/blob/master/tests/test_class_oelint_task_docstrings.py